pom.xml

```diff
@@ -16,21 +16,32 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
I've gone ahead and updated core and many of the dependent modules to their latest release version.  Ideally, we'd test all of these and deploy.